packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3211,6 +3211,7 @@ export interface VideoGalleryProps {
     localParticipant: VideoGalleryLocalParticipant;
     localVideoCameraCycleButtonProps?: LocalVideoCameraCycleButtonProps;
     localVideoViewOptions?: VideoStreamOptions;
+    maxPinnedRemoteVideoTiles?: number;
```

Comment:
I am not sure if we ever discussed this prop as part of the public API changes.
Do we really need to introduce this?
I was under the impression that VideoGallery was going to enforce default of 4

Reply:
If its helpful I really like this prop! In there a reason to limit to 4?

Reply:
I can hardcode max limit of 4 pinned remote video tiles to disable the pin menu item just to avoid the API change. And we can add this prop in the future from customer signals by reverting the commit hash on this branch.